coord/src/rmr/rmr.c

```diff
@@ -442,7 +442,7 @@ int MR_Map(struct MRCtx *ctx, MRReduceFunc reducer, MRCommandGenerator cmds, boo
     RedisModule_Assert(!ctx->bc);
     ctx->bc = RedisModule_BlockClient(ctx->redisCtx, unblockHandler, timeoutHandler,
       redisMajorVesion < 5 ? (void (*)(RedisModuleCtx *, void *))freePrivDataCB : freePrivDataCB_V5,
-      timeout_g);
+      0); // timeout_g);
```

Comment:
Explain?

Reply:
see my [reply here](https://github.com/RediSearch/RediSearch/pull/3853#discussion_r1349698820). It was also requested by @DvirDukhan so it is clear that this is what currently happening. We can discuss it in person.